requirements/dev.txt

```diff
@@ -19,11 +19,6 @@ appdirs==1.4.4
     #   virtualenv
 async-timeout==4.0.0a3
     # via -r requirements/base.txt
-attrs==20.3.0
-    # via
-    #   -r requirements/lint.txt
-    #   flake8-pyi
-    #   pytest
```

Comment:
Don't edit this file, it's managed automatically. In this case, `attrs` comes in as a transitive dependency as visible in the comment.